### DISCUSSION TITLE
Page type(s)

### DISCUSSION THREAD

**Discussion by bbcakes4real** [WikiTransformationProject/wikitraccs-releases#18] [General]
What type of [SharePoint pages](https://learn.microsoft.com/en-us/sharepoint/dev/solution-guidance/sharepoint-pages-and-the-page-model) does the wikitraccs tool create?

**Comment by heinrich-ulbricht** [comment]
@bbcakes4real WikiTraccs creates SharePoint Online modern pages (no classic pages). It uses a content type derived from the standard Site Page content type to store metadata.

(Sorry for not getting back to you earlier, notifications were broken :( )
